package com.revature.exceptions;

public class UserNotFoundException extends RuntimeException {

}
